curriculum/challenges/english/25-front-end-development/workshop-flappy-penguin/61967e74a8e3690ab6292daa.md

```diff
@@ -36,7 +36,7 @@ assert.include(['linear-gradient(45deg,rgb(118,201,255),rgb(247,255,222))', 'rgb
 ```html
 <!DOCTYPE html>
 <html lang="en">
-  
+
```

Comment:
there are again many of these changes, it makes review harder, above all if they are not in files you actually changed